blinkit/page-1.html

```diff
@@ -0,0 +1,328 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <title>Blinkit Landing Page</title>
+    <link rel="stylesheet" href="styles/styles.css">
+</head>
+
+<body>
+    <nav class="navigation-bar">
+        <img id="blinkit-logo" src="images/Screenshot 2023-01-30 at 6.12.11 PM.png">
```

Comment:
1. We should name the images better so that it is readable.
2. We should use alt here,  in case there is an issue with the image.